test/filter.ts

```diff
@@ -50,7 +59,7 @@ describe('AsyncRay', () => {
 
       should(outputArray)
         .instanceOf(Array)
-        .empty();
```

Comment:
`aFilter` returns now an new instance of `AsyncArray` which has the (private) property `input`, thus `empty()` will not work at this point anymore